Src/WitsmlExplorer.Frontend/components/ContentViews/JobsView.tsx

```diff
@@ -146,13 +146,19 @@ export const JobsView = (): React.ReactElement => {
                 : jobInfo.status,
             cancel:
               jobInfo.isCancelable === true && jobInfo.status === "Started" ? (
-                <StyledButton
-                  key="downloadall"
-                  variant="outlined"
-                  onClick={() => cancelJob(jobInfo.id)}
-                >
-                  <Icon name="clear" />
-                </StyledButton>
+                <EdsProvider density="compact">
```

Comment:
I noticed that the size of the button is not responsive to the density mode. Maybe we can conditionally set the button size for different density modes?

Reply:
Done